include/seqan3/alignment/pairwise/edit_distance_unbanded.hpp

```diff
@@ -267,7 +270,7 @@ class pairwise_alignment_edit_distance_unbanded
             _best_score = _score;
         }
 
-        word_type vp0{~static_cast<word_type>(0)};
+        word_type vp0{static_cast<word_type>(~0)};
```

Comment:
Interesting. Is the compiler expanding  the bits to the word_size? I mean `0` would be just int right?

Reply:
It seems that `~` is a operator on `int`. It gave me at least some problems for `int8_t/uint8_t`.

`0` is of type `int`.